src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/View.tsx

```diff
@@ -130,120 +130,99 @@ export const SettingsView = (props: Props) => {
             </>
           )}
           <hr />
-          {!props.enabledFeatureFlags.includes("MonitorAccountDeletion") ? (
-            <div className={styles.deactivateSection}>
```

Comment:
Oh, looks like I forgot my comment - for an extra thorough cleanup, you could also remove the reference to this class in https://github.com/mozilla/blurts-server/blob/52ea336a9da4c5ef95aa5e6384b9e95e0e520567/src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/View.module.scss#L58

Reply:
[chore: remove ref for deactivateSection](https://github.com/mozilla/blurts-server/pull/4586/commits/a1fdf1c795ea5b6c4403e346ef2b392fecf8aa95)